src/Game/zScript.h

```diff
@@ -1,6 +1,25 @@
 #ifndef ZSCRIPT_H
 #define ZSCRIPT_H
 
+#include <types.h>
+#include "../Core/x/xBase.h"
+#include "../Core/x/xDynAsset.h"
 
+struct xScriptAsset : xDynAsset
+{
+    // There is definitely more here, but idk what.
+};
+
+struct zScript : xBase
+{
+    xScriptAsset* asset;
+    // I'm 90% sure there is more here, but idk what.
+};
```

Comment:
Again, also in the dwarf data: https://github.com/pslehisl/bfbbdecomp/blob/master/dwarf/SB/Game/zScript.cpp#L1411-L1418

Please also name it `_zScript` to be consistent with the names that the developers gave it:

```
PS C:\Users\mattp\Documents\bfbbdecomp> c++filt.exe zScriptReset__FP8_zScript
zScriptReset(_zScript *)
```
It says the type has an underscore.